app/services/follows/delete_cached.rb

```diff
@@ -0,0 +1,33 @@
+module Follows
+  class DeleteCached
+    def self.delete(follower, followable_type, followable_id)
+      new(follower, followable_type, followable_id).delete
+    end
+
+    def initialize(follower, followable_type, followable_id)
+      @follower = follower
+      @followable_type = followable_type
+      @followable_id = followable_id
+    end
+
+    def delete
+      return false unless follower
+
+      cache_key = "user-#{follower.id}-#{follower.updated_at.rfc3339}/is_following_#{followable_type}_#{followable_id}"
+      Rails.cache.delete(cache_key)
+    end
+
+    private
+
+    attr_accessor :follower, :followable_type, :followable_id
+
+    def followable
```

Comment:
I can't find a place in the code where this method is called, it comes from `Follows::CheckCached` as it needs it here:

https://github.com/forem/forem/blob/b7ff9aadd1274124e19281ba247486d9c3e33a2a/app/services/follows/check_cached.rb#L18

Reply:
woops! yes I got a bit too keen with some copy/paste there 😅